src/server/graphql/models/Meeting/meetingMutation.js

```diff
@@ -1,20 +1,20 @@
 import {Meeting} from './meetingSchema';
-import r from '../../../database/rethinkdriver';
-import {isLoggedIn} from '../authorization';
+import r from '../../../database/rethinkdriver'; // eslint-disable-line id-length
+// import {isLoggedIn} from '../authorization';
 import uuid from 'node-uuid';
 import {
-  GraphQLBoolean,
+  // GraphQLBoolean,
   GraphQLString,
-  GraphQLObjectType,
+  // GraphQLObjectType,
   GraphQLNonNull,
   GraphQLID,
-  GraphQLList
+  // GraphQLList
 } from 'graphql';
 
 export default {
   createMeeting: {
     type: Meeting,
-    async resolve(source, args, {rootValue}) {
+    async resolve(source, args, {rootValue}) { // eslint-disable-line no-unused-vars
       const {authToken} = rootValue;
       // isLoggedIn(authToken);
```

Comment:
we can remove it for now, but chances are we'll want to make sure whoever calls `createMeeting` is logged in first, right? First we'll need to make a good ER model though. and then relational schema. and then do fun query mapping... then we can come back to this :-)